assets/app/styles/_log.less

```diff
@@ -100,14 +90,16 @@
   }
 }
 .log-line-number {
-  .flex(@columns: 0 0 60px); // set width of number column
+  .unselectable();
+  border-right: 1px lighten(@log-bg-color, 10%) solid;
+  min-width: 60px;
```

Comment:
do we actually need this much space, or can we shrink it further? might want to set `white-space: nowrap;` as well if you do